pkgs/os-specific/linux/fuse/common.nix

```diff
@@ -43,34 +43,34 @@ in stdenv.mkDerivation rec {
     then [ meson ninja pkg-config ]
     else [ autoreconfHook gettext ];
 
-  outputs = [ "out" ] ++ lib.optional isFuse3 "common";
+  outputs = [ "bin" "out" "dev" "man" ] ++ lib.optional isFuse3 "common";
 
   mesonFlags = lib.optionals isFuse3 [
     "-Dudevrulesdir=/udev/rules.d"
     "-Duseroot=false"
+    # `fuserconf_path` is pinned to `/etc/fuse.conf` in `./fuse3-install.patch`,
+    # we are free to modify this option now.
+    "-Dsysconfdir=${placeholder "common"}/etc"
   ];
 
   preConfigure = ''
-    export MOUNT_FUSE_PATH=$out/sbin
-    export INIT_D_PATH=$TMPDIR/etc/init.d
-    export UDEV_RULES_PATH=$out/etc/udev/rules.d
-
     # Ensure that FUSE calls the setuid wrapper, not
     # $out/bin/fusermount. It falls back to calling fusermount in
     # $PATH, so it should also work on non-NixOS systems.
     export NIX_CFLAGS_COMPILE="-DFUSERMOUNT_DIR=\"/run/wrappers/bin\""
 
     substituteInPlace lib/mount_util.c --replace "/bin/" "${util-linux}/bin/"
-    '' + (if isFuse3 then ''
-      # The configure phase will delete these files (temporary workaround for
-      # ./fuse3-install_man.patch)
-      install -D -m444 doc/fusermount3.1 $out/share/man/man1/fusermount3.1
-      install -D -m444 doc/mount.fuse3.8 $out/share/man/man8/mount.fuse3.8
-    '' else ''
```

Comment:
Why it was removed I don't see replacement

Reply:
They used to be together with something inside that `fuse3-install_man.patch`. But somehow it's upstreamed, and now that files are installed correctly without manual tweaks.